bindings_ffi/src/mls.rs

```diff
@@ -1144,6 +1144,8 @@ mod tests {
     }
 
     #[tokio::test(flavor = "multi_thread", worker_threads = 5)]
+    // This one is flaky for me. Passes reliably locally and fails on CI
```

Comment:
yeah this is the one that fails because of the timeout (found during the refactor). It passes locally since CI is just slower. using `async_barrier` for sync points instead of `tokio::sleep` should fix it.

not sure if a ticket yet but should be, will make it if not

Reply:
I don't think this one has a ticket. You mind making one